integration/integration_test.go

```diff
@@ -43,7 +47,7 @@ func testWithQEMU(t *testing.T, uinitPkg string) (string, *qemu.QEMU) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	pkgs = append(pkgs, uinitPkg)
+	pkgs = append(pkgs, path.Join("github.com/u-root/u-root/integration/testdata", uinitName, "uinit"))
```

Comment:
use filepath, not path.

Reply:
This is actually a case where you want to use `path` and not `filepath`. This is a Go package path, not a file path. It always uses `/` slashes, even on Windows.

Reply:
oops :-)